include/faunus/space.h

```diff
@@ -79,6 +96,16 @@ namespace Faunus
           return sum;
       }
 
+      Group* get(int i)
```

Comment:
To get the i'th group, wouldn't one just call `Space::groupList()[i]`? If this is not what this function is doing, the name is confusing.